openapi.yml

```diff
@@ -1010,10 +1010,9 @@ components:
             type: string
         metadata-object:
           $ref: "#/components/schemas/metadata-object"
-
       required:
-        - breadcrumbs
-        - metadata
+        - breadcrumb
+        - metadata-object
```

Comment:
oops, yes this is correct. however the key is actually `breadcrumbs`, can you change the name under properties?